examples/admin_policy/example_server/dynamic_policy_server.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
```

Comment:
Not sure how much difference this is compared to `policy_server.py`. Is this necessary?

Reply:
This is kept to keep the unit test work, in unit test we have to load policies dynamically in the server to test different scenarios but we remove it from our user-facing example (which will be referenced in https://docs.skypilot.co/en/latest/cloud-setup/policy.html#host-admin-policy-as-server) to keep the doc simple.